configure

```diff
@@ -425,6 +425,11 @@ intl_optgroup.add_option('--with-icu-source',
     dest='with_icu_source',
     help='Intl mode: optional local path to icu/ dir, or path/URL of icu source archive.')
 
+parser.add_option('--with-ltcg',
+    action='store_true',
+    dest='with_ltcg',
+    help='Use Link Time Code Generation. This feature is only available on Windows.')
```

Comment:
I would suggest flipping the switch to `--disable-windows-ltcg` similar to:

Reply:
I wanted to keep this in line with Linux - there was PR to add `--with-lto`, at some point, when it will be ready that can be merged into this option

Reply:
The difference is that if we want LTCG to be the default, it makes more sense to have an opt-out `--without` flag so the call to `python configure` could stay simple.

BTW I meant to refeance: https://github.com/nodejs/node/blob/23efa8cc0692e690fd5138f5b93ccfd4e47c2f58/configure#L525-L528